profiler/test/Datadog.Profiler.IntegrationTests/Datadog.Profiler.IntegrationTests.csproj

```diff
@@ -36,6 +36,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="Google.Protobuf" Version="3.20.1" />
```

Comment:
Does this work on alpine + arm64? I know I had all sorts of issues with GRPC, but I can't remember if that was the grpc-tool or protobuf lib